docs/profiles/profile-single-tenant-cluster.md

```diff
@@ -0,0 +1,93 @@
+# Kubernetes Multi-tenancy profile: Single Tenant Cluster
+
+In order to define a secure multi-tenant cluster, we must first agree upon a baseline for a secure single tenant cluster. 
+
+## Overall functional model summary
+
+1. A **Tenant** is defined as a team of users/ identities that shall have exclusive use of one or more resources of a Kubernetes cluster in parallel with users of other tenants using their own separate and isolated resources within the same cluster. We will interchangeably use the terms \"users\" or \"members\" of a tenant.
```

Comment:
Technically, as called out in [Tenant Concept in Kubernetes](https://docs.google.com/document/d/1ddx7UAEPKFPldBh_diksYO4WZXSHDUhm-e6hyNNGYVU/edit#heading=h.c0uts5ftkk58), defining Tenant as users is ambiguous and problematic. I think I understand overall what this paragraph means and I agree with that. However, is it possible avoid explicitly defining a tenant as a group of users? I recommend saying "A tenant is defined as a group of resources to be exclusively used by a team of users/identities" and avoid the interchangeable use of terms of users/members with tenants. The point is to define tenant from resource perspective instead from users perspective to reduce the ambiguity. My two cents.